graylog2-web-interface/src/components/content-packs/ContentPackInstallastions.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
This file has a typo in its name:

`ContentPackInstallastions.test.jsx`
`ContentPackInstallations.test.jsx`